packages/exmg-form/exmg-form.ts

```diff
@@ -140,6 +157,52 @@ export class ExmgForm extends LitElement {
     }
   }
 
+  protected firstUpdated() {
+    this.baseValues = this.serializeForm();
+  }
+
+  private handleOnChange(_e: Event): void {
+    if (this.dirty) {
+      return;
+    }
+    this.dirty = true;
+    this.dispatchEvent(
+      new CustomEvent('dirty', {
```

Comment:
I think it is better to always specify bubbles and composed when using CustomEvent